### ISSUE TITLE
`.validate()` false-positive

### ISSUE THREAD

**Issue by sindresorhus** [NaturalIntelligence/fast-xml-parser#327]
* [x] Are you running the latest version?
* [x] Have you included sample input, output, error, and expected output? 
* [x] Have you checked if you are using correct configuration?
* [x] Did you try online [tool](https://naturalintelligence.github.io/fast-xml-parser/)?

### Input

```xml
<svg></svg> foo bar
```

### Code
```js
const parser = require('fast-xml-parser');

console.log(parser.validate('<svg></svg> foo bar'));
```

### Output

```
true
```

### expected data

Expected it to be false or an object describing the error.

**Would you like to work on this issue?**

- [ ] Yes
- [x] No

[Bookmark](https://github.com/NaturalIntelligence/fast-xml-parser/stargazers) this repository for further updates.
